libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -13,8 +13,18 @@ async function getKey(product) {
   return keyMatch[0]?.key;
 }
 
+async function getECID() {
+  let ecid = null;
+  if (window.alloy) {
+    await window.alloy('getIdentity').then((data) => {
+      ecid = data?.identity?.ECID;
+    }).catch((err) => window.lana.log(err));
```

Comment:
I think we should categorize the error and provide the appropriate labels, e.g. https://github.com/adobecom/milo/blob/a856073ad5b4f0d72642be05e56e6aee9a06c4d8/libs/blocks/quiz-results/quiz-results.js#L23

Reply:
Sure, added the same.